x/faucet/genesis.go

```diff
@@ -0,0 +1,21 @@
+package faucet
+
+import (
+	"github.com/Sifchain/sifnode/x/faucet/keeper"
+	"github.com/Sifchain/sifnode/x/faucet/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// InitGenesis initialize default parameters
+// and the keeper's address to pubkey map
+func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
```

Comment:
We do need these functions implemented , otherwise the upgrades will fail.
If not now ,there should atleast be a task for it on asana

Reply:
Will sync up with you on Monday to get some clarification around this and craft an asana ticket. Can this fit in a separate PR?